examples/plot_unified_validation_scheme.py

```diff
@@ -0,0 +1,344 @@
+"""
+Unified Validation Scheme
+========================================
+
+This tutorial shows you how to properly train, tune and test your deep learning
+models with Braindecode. We will use the BCIC IV 2a dataset as a showcase example,
+however this scheme holds for any supervised trial-based decoding setting.
+As this tutorial will include additional parts of code like loading and preprocessing,
+defining a model, etc. which are not exclusive to this page (compare `Cropped Decoding
+Tutorial <./plot_bcic_iv_2a_moabb.html>`__), feel free to skip these parts.
```

Comment:
I would link to trialwise example. This also fixes the link to the file.
```suggestion
defining a model, etc. which are not exclusive to this page (compare `Trialwise Decoding
Tutorial <./plot_bcic_iv_2a_moabb_trial.html>`__), feel free to skip these parts.
```